api/management/commands/loaddata.py

```diff
@@ -0,0 +1,17 @@
+import os
+import csv
+from django.core.management.base import BaseCommand
+from api.models import Disease
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        path = os.path.join('data', 'diseases.tsv')
+        with open(path, 'r') as diseases_file:
+            diseases_rows = csv.reader(diseases_file, delimiter='\t')
+            # Skip header
+            next(diseases_rows)
+            for row in diseases_rows:
+                acronym = row[0]
```

Comment:
My rule of thumb is to never refer to columns in a dataframe positionally. It's prone to breaking and less explicit. Check out `csv.DictReader` as an alternative.